### DISCUSSION TITLE
[Backend] Go (net/http) standard library

### DISCUSSION THREAD

**Discussion by 0xdod** [realworld-apps/realworld#821] [Added implementations]
Repo: https://github.com/0xdod/go-realworld

**Comment by geromegrignon** [comment]
Hey @0xdod, thanks for your contribution!
Is your implementation completed?

**Reply by 0xdod** [reply]
Yes it is

**Reply by geromegrignon** [reply]
Awesome, can you submit it [here](https://codebase.show/projects/realworld/implementations/submit) so it'll be included in the official list?

**Reply by 0xdod** [reply]
Done

**Reply by geromegrignon** [reply]
Thanks, approved!

**Reply by mvila** [reply]
To make the implementation look better in [the list](https://codebase.show/projects/realworld?category=backend&language=go), I've changed the libraries/frameworks field from "None" to "Go's net/http library". @0xdod, feel free to edit it if you have a better idea.

**Reply by 0xdod** [reply]
I think that's more appropriate, thanks for the feedback
